Integrations/Snowflake/Snowflake.py

```diff
@@ -0,0 +1,392 @@
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get('last_fetched_data_timestamp')
+    first_fetch = False
+
+    # Handle first time fetch, fetch incidents retroactively
+    if not last_fetch:
+        first_fetch = True
+        last_fetch, _ = parse_date_range(FETCH_TIME, to_timestamp=True)
+    args = {'rows': MAX_ROWS, 'query': FETCH_QUERY}
+
+    column_descriptions, data = snowflake_query(args)
+    # convert the data/events to demisto incidents
+    incidents = []
+    for row in data:
+        incident = row_to_incident(column_descriptions, row)
+        incident_timestamp = incident.get('timestamp')
+
+        # Update last run and add incident if the incident is newer than last fetch
+        if incident_timestamp and incident_timestamp >= last_fetch:
+            if first_fetch:
+                last_fetch = incident_timestamp
+                incidents.append(incident)
+                first_fetch = False
+            elif len(incidents) >= 1 and incident.get('rawJSON') != incidents[-1].get('rawJSON'):
+                last_fetch = incident_timestamp
+                incidents.append(incident)
+
+    demisto.setLastRun({'last_fetched_data_timestamp': last_fetch})
+    demisto.incidents(incidents)
+
+
+def snowflake_query(args):
+    connection = get_connection(args)
+    if connection:
+        try:
+            query = args.get('query')
+            cur = connection.cursor(snowflake.connector.DictCursor)
+            cur.execute(query)
+            rows = args.get('rows')
+            if rows:
+                rows = int(rows)
+            if rows and rows > MAX_ROWS:
+                rows = MAX_ROWS
+            results = cur.fetchmany(rows)
+            if results:
+                return cur.description, results
+            else:
+                return_error('No data found matching the query')
+        except snowflake.connector.errors.ProgrammingError as e:
+            return_error(str(e))
+        finally:
+            if cur:
+                cur.close()
+            connection.close()
+    else:
+        return_error('Unable to connect')
+
+
+def snowflake_query_command():
+    args = demisto.args()
+    query = args.get('query')
+    db = args.get('database') if args.get('database') else DATABASE
+    schema = args.get('schema') if args.get('schema') else SCHEMA
+    col_descriptions, results = snowflake_query(args)
+    results = format_to_json_serializable(col_descriptions, results)
+
+    entry_context = {
+        'Database': db,
+        'Schema': schema,
+        'Query': query,
+        'Result': results
+    }
+    columns = argToList(args.get('columns'))
+    human_readable = tableToMarkdown(query, results, columns)
+    demisto_transform = 'Snowflake(val.Query && val.Query === obj.Query'
+    demisto_transform += ' && val.Database && val.Database === obj.Database'
+    demisto_transform += ' && val.Schema && val.Schema === obj.Schema)'
+
+    demisto.results({
```

Comment:
use `return_outputs`

Reply:
I switched `demisto.results` for `return_outputs` for the `snowflake-query` command as per your request but for the `snowflake-update` command I'm only returning a string to the war room and to do that with the `return_outputs` wrapper it would look like 
```
return_outputs(outputs={}, readable_output='Operation executed successfully')
```
as opposed to 
```
demisto.results('Operation executed successfully')
```
which I think actually looks cleaner in this particular case.  

---
Do you still want me to switch it there as well?